src/components/authentication/LoginPage.vue

```diff
@@ -1,30 +1,62 @@
 <template>
 	<div class="login-page">
+		<div class="login-page__alert">
+			<v-alert
+				:value="true"
+				v-if="status.success"
+				color="success"
+				icon="check_circle"
+				outline
+			>
+				You have successfully logged in!
+			</v-alert>
+		</div>
 
-		<div class="login-page__spacer"/>
+		<div class="login-page__alert">
+			<v-alert
+				:value="true"
+				v-if="status.error"
+				color="error"
+				icon="warning"
+				outline
+			>
+				Wrong email or password!
```

Comment:
You can get this message from response

Reply:
yes but no, now a can't get response from incorrect get-requests. need to make a catch for that